packages/mapviewer/src/modules/infobox/components/FeatureDetail.vue

```diff
@@ -181,11 +181,10 @@ function getIframeHosts(value) {
 :global(.t_list) {
     width: 100%;
 }
-td {
+:global(td) {
     vertical-align: top;
 }
-
-td.cell-left {
+:global(td.cell-left) {
```

Comment:
although `.cell-left` is scoping this rule better than the one above, I'd also would find it better if we'd use `.htmlpopup-container` class to reduce the scope